### DISCUSSION TITLE
launch crashes with no errors, any idea how to fix?

### DISCUSSION THREAD

**Discussion by Shamunali** [vladmandic/sdnext#1163] [Q&A]
![image](https://github.com/vladmandic/automatic/assets/134617046/2b2d67c2-7d3f-45ae-bac5-3ec85790a8a1)

It worked just fine and suddenly it all crashed. The console and the Server with it.
Since then its the same, i launch it, its written like in the picture and crashes.
Had to take a photo with my phone so i could catch the whats written before it crashes.
Any idea whats the problem how to fix it?

**Comment by Yodrik** [comment]
If you run with the cmd arg, --debug [(other arguments like this are described in the development update)](https://github.com/vladmandic/automatic/discussions/99#discussion-5078542) you can see additional info that may be relevant. Also if it is a bother to have the window disappear, just be sure to do the launching in a cmd/powershell window. If it is crashing it will just send you back out to the command line,

**Comment by vladmandic** [comment]
marking as invalid until information can be provided - like @Yodrik said, launch from command line instead of using a shortcut so window does not close automatically. use `--debug` flag and post log here as text - not a camera picture and not a screen snapshot.
